processors/pubmed/extractors.py

```diff
@@ -42,6 +52,38 @@ def extract_publications(record):
         'date': record['article_date'],
         # ---
         'slug': slug,
-    })
+        'registry_ids': registry_ids,
+        'identifiers': identifiers,
+    }
+
+
+def extract_document(record):
+    return {
+        'name': record['article_title'],
+        'source_url': record['meta_source'],
+    }
+
+
+def extract_document_category(record):
+    return base.config.DOCUMENT_CATEGORIES['journal_article']
+
+
+def extract_trial(record):
+    pubmed_id = base.helpers.safe_prepend('PUBMED', record['pmid'])
+    identifiers = []
+    ids = base.helpers.find_list_of_identifiers(pubmed_id)
```

Comment:
Use named parameter here to increase readability: `base.helpers.find_list_of_identifiers(text=pubmed_id)`